cmd/gaia/app/sim_test.go

```diff
@@ -93,9 +96,13 @@ func appStateFromGenesisFileFn(r *rand.Rand, accs []simulation.Account, genesisT
 	return genesis.AppState, newAccs, genesis.ChainID
 }
 
-func appStateRandomizedFn(r *rand.Rand, accs []simulation.Account, genesisTimestamp time.Time) (json.RawMessage, []simulation.Account, string) {
+// TODO refactor out random initialization code to the modules
```

Comment:
Should we open an issue here?

Reply:
yup I'll do that

Reply:
https://github.com/cosmos/cosmos-sdk/issues/4298